### ISSUE TITLE
Documentation states wrong configuration prefix.

### ISSUE THREAD

**Issue by wondee** [spring-projects/spring-security-oauth2-boot#200]
### Summary

The Documenation states that the prefix for activating the `OAuth2ResourceServerConfiguration` is `spring.security.oauth.resource.*`, but in the `getMatchOutcome(..)` method the prefix is set like this `String prefix = "security.oauth2.resource.";`. 

### Configuration

Added `@EnableResourceServer` and add configuration like: 

```yaml

spring: 
  security:
    oauth2:
      resource:
        jwt:
          key-value: somekey

```

### Sample

See: https://docs.spring.io/spring-security-oauth2-boot/docs/current/reference/html/boot-features-security-oauth2-resource-server.html#oauth2-boot-resource-server-jwt-single-key

**Comment by jzheaux** [comment]
Thanks for the report, @wondee! Would you be interested in submitting a PR?

**Comment by wondee** [comment]
Sure, what shall be fixed? Documentation or Placeholder Prefix?

**Comment by jzheaux** [comment]
Good question, @wondee. Let's make sure the documentation matches what the code already does.

**Comment by jzheaux** [comment]
Fixed via bbaf06998088044d207800869a3f38ad0304d4f6
